src/components/App/App.tsx

```diff
@@ -17,7 +17,7 @@ const parseMass = (value: number, exponent: number) => {
 
 
 class App extends React.Component<{}, AllData> {
-  constructor(props: any) {
+  constructor(props: {}) {
```

Comment:
Can't believe we missed this one! Thanks for fixing it.

Reply:
Likewise!